client/my-sites/stats/site.jsx

```diff
@@ -406,20 +408,28 @@ class StatsSite extends Component {
 					</div>
 				</div>
 
-				<div className="stats-content-promo">
-					<PromoCardBlock
-						productSlug="wordpress-seo-premium"
-						impressionEvent="calypso_stats_wordpress_seo_premium_banner_view"
-						clickEvent="calypso_stats_wordpress_seo_premium_banner_click"
-						headerText={ translate( 'Increase site visitors with Yoast SEO Premium' ) }
-						contentText={ translate(
-							'Purchase Yoast SEO Premium to ensure that more people find your incredible content.'
-						) }
-						ctaText={ translate( 'Learn more' ) }
-						image={ wordpressSeoIllustration }
-						href={ `/plugins/wordpress-seo-premium/${ slug }` }
-					/>
+				<div className="stats-traffic-promo-container">
```

Comment:
```suggestion
				<div className="stats__promo-container">
```

According to the [naming guide of BEM](https://getbem.com/naming/), could we have this container with a more comprehensive name and append the modifier `--traffic` when necessary?

Reply:
Thanks for the BEM link, Dognose. I reviewed that and the [Calypso CSS guidelines](https://github.com/Automattic/wp-calypso/blob/trunk/docs/coding-guidelines/css.md) and I think I've got it right now. Let me know if you see anything else that isn't a best practice!